config.yaml

```diff
@@ -52,8 +52,17 @@ uenvs:
         gh200: v0.9/gh200
       deploy:
         eiger: [zen2]
-        santis: [gh200]
```

Comment:
This is a change related to `arbor`. Is it indended to have it here?

Reply:
It was a drive-by fix